src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/jpa/repository/PersonRepositoryIntTest.kt

```diff
@@ -53,20 +60,6 @@ class PersonRepositoryIntTest : IntegrationTestBase() {
     assertThat(personEntity).isNull()
   }
 
-  @Test
```

Comment:
this doesn't appear to test anything which is not covered by `should return correct person records with matching pnc number`